services/src/main/java/org/apache/druid/server/AsyncQueryForwardingServlet.java

```diff
@@ -857,8 +858,9 @@ public void onFailure(Response response, Throwable failure)
       failedQueryCount.incrementAndGet();
 
       // As router is simply a proxy, we don't make an effort to construct the error code from the exception ourselves.
-      // We rely on broker to set this for us if the error occurs downstream.
-      emitQueryTime(requestTimeNs, false, sqlQueryId, queryId, response.getStatus());
+      // We rely on broker to set this for us if the error occurs downstream. Otherwise, if there's a router/client error, we log this as an ISE.
```

Comment:
ISE might be confused with the Druid `ISE` (flavor of IllegalStateException).

```suggestion
      // We rely on broker to set this for us if the error occurs downstream.
      // Otherwise, if there's a router/client error, we log this as an unknown error.
```